### ISSUE TITLE
Performance on small datasets?

### ISSUE THREAD

**Issue by addiu** [ajbrock/BigGAN-PyTorch#19]
Hi,
does anyone know how BigGAN performs on small datasets? e.g. the birds dataset (Caltech-UCSD Birds-200-2011)?  In such dataset, we only have about 50 images per class.
I have tried to run BigGAN on the birds dataset, however, the result doesn't look very nice (worse than baseline models)... since BigGAN is a large model, does it need a large dataset to perform well?

**Comment by ajbrock** [comment]
In generative modeling, as in any machine learning application, the capacity of your model should be commensurate with the complexity of the task. 

You don't need big models or big batches for CIFAR because the dataset is minimally complex and a small batch likely provides sufficiently accurate gradients; on an ImageNet scale task or larger, however, there is a lot more information the model needs to deal with, and a lot more modes to cover. For smaller datasets, I'd recommend using EMA and the BigGAN architectural tricks (shared embeddings, hierarchical latents, truncation and maybe ortho reg) but you probably don't want an actual Big model for this problem. 

Learning from scarce data is an open research question in general, and there's tons of work on low-shot learning (and plenty of generative models trained on CUB-200). A recent paper that may be of interest to you is [Image Generation from Small Datasets via Batch Statistics Adaptation](https://arxiv.org/abs/1904.01774) by Atsuhiro Noguchi and Tatsuya Harada.
